Conflict between Glassfish/Payara and JClouds dependence.

# Description

---

If I add the dependency of JClouds with Maven in the project:

``` xml
        <dependency>
            <groupId>org.apache.jclouds</groupId>
            <artifactId>jclouds-all</artifactId>
            <version>1.9.2</version>
        </dependency>
```

The application is not deployed, I get the error:

``` java
Launching GlassFish on Felix platform
out 30, 2016 11:47:15 AM com.sun.enterprise.glassfish.bootstrap.osgi.BundleProvisioner createBundleProvisioner
INFORMAÇÕES: Create bundle provisioner class = class com.sun.enterprise.glassfish.bootstrap.osgi.BundleProvisioner.
Registered com.sun.enterprise.glassfish.bootstrap.osgi.EmbeddedOSGiGlassFishRuntime@38c2d2d7 in service registry.
out 30, 2016 11:47:26 AM com.sun.logging.LogDomains getResourceBundle
INFORMAÇÕES: Cannot find the resource bundle for the name com.sun.logging.enterprise.system.container.ejb for class org.glassfish.kernel.javaee.MEJBService using org.glassfish.main.core.javaee-kernel [134]
#!## LogManagerService.postConstruct : rootFolder=C:\Ferramentas\payara41\glassfish
#!## LogManagerService.postConstruct : templateDir=C:\Ferramentas\payara41\glassfish\lib\templates
#!## LogManagerService.postConstruct : src=C:\Ferramentas\payara41\glassfish\lib\templates\logging.properties
#!## LogManagerService.postConstruct : dest=C:\Ferramentas\payara41\glassfish\domains\domain1\config\logging.properties
Informações:   Running Payara Version: Payara Server  4.1.1.163 #badassfish (build 215)
Informações:   Server log file is using Formatter class: com.sun.enterprise.server.logging.ODLLogFormatter
Informações:   Registered org.glassfish.ha.store.adapter.cache.ShoalBackingStoreProxy for persistence-type = replicated in BackingStoreFactoryRegistry
Informações:   Authorization Service has successfully initialized.
Informações:   Registered fish.payara.ha.hazelcast.store.HazelcastBackingStoreFactoryProxy for persistence-type = hazelcast in BackingStoreFactoryRegistry
Informações:   Registered Hazelcast BackingStoreFactory with persistence-type = hazelcast
Informações:   Realm [admin-realm] of classtype [com.sun.enterprise.security.auth.realm.file.FileRealm] successfully created.
Informações:   Realm [file] of classtype [com.sun.enterprise.security.auth.realm.file.FileRealm] successfully created.
Informações:   Realm [certificate] of classtype [com.sun.enterprise.security.auth.realm.certificate.CertificateRealm] successfully created.
Informações:   Grizzly Framework 2.3.25 started in: 119ms - bound to [/0.0.0.0:8080]
Informações:   Grizzly Framework 2.3.25 started in: 5ms - bound to [/0.0.0.0:8181]
Informações:   Grizzly Framework 2.3.25 started in: 20ms - bound to [/0.0.0.0:4848]
Informações:   Grizzly Framework 2.3.25 started in: 6ms - bound to [/0.0.0.0:3700]
Informações:   Payara Server  4.1.1.163 #badassfish (215) startup time : Felix (12.295ms), startup services(1.482ms), total(13.777ms)
Informações:   Cleaning JarFileFactory Cache to prevent jar FD leaks
Informações:   Grizzly Framework 2.3.25 started in: 4ms - bound to [/0.0.0.0:7676]
Informações:   Registered com.sun.enterprise.glassfish.bootstrap.osgi.EmbeddedOSGiGlassFishImpl@782bf610 as OSGi service registration: org.apache.felix.framework.ServiceRegistrationImpl@62d73ead.
Informações:   C:\Ferramentas\payara41\glassfish\domains\domain1\autodeploy\bundles does not exist, please create it.
Informações:   HV000001: Hibernate Validator 5.1.2.Final
Informações:   Grizzly Framework 2.3.25 started in: 12ms - bound to [/0.0.0.0:8181]
Informações:   JMXStartupService has started JMXConnector on JMXService URL service:jmx:rmi://douglas-hp:8686/jndi/rmi://douglas-hp:8686/jmxrmi
Informações:   Java security manager is disabled.
Informações:   Entering Security Startup Service.
Informações:   Loading policy provider com.sun.enterprise.security.provider.PolicyWrapper.
Informações:   Security Service(s) started successfully.
Informações:   Created HTTP listener http-listener-1 on host/port 0.0.0.0:8080
Informações:   Created HTTP listener http-listener-2 on host/port 0.0.0.0:8181
Informações:   Created HTTP listener admin-listener on host/port 0.0.0.0:4848
Informações:   Created virtual server server
Informações:   Created virtual server __asadmin
Informações:   Setting JAAS app name glassfish-web
Informações:   Virtual server server loaded default web module 
Informações:   WELD-000900: 2.3.5 (Final)
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-001125: Illegal bean type com.google.common.base.Function<org.jclouds.blobstore.domain.PageSet<org.jclouds.openstack.swift.domain.ObjectInfo>, org.jclouds.blobstore.domain.PageSet<? extends org.jclouds.blobstore.domain.StorageMetadata>> ignored on [EnhancedAnnotatedTypeImpl] public @Singleton class org.jclouds.openstack.swift.blobstore.functions.ContainerToResourceList
Informações:   WELD-001125: Illegal bean type com.google.common.base.Supplier<java.util.Set<? extends org.jclouds.compute.domain.Hardware>> ignored on [EnhancedAnnotatedTypeImpl] public @Singleton class org.jclouds.gogrid.compute.suppliers.GoGridHardwareSupplier
Informações:   WELD-001125: Illegal bean type com.google.common.base.Function<java.util.Set<org.jclouds.s3.domain.BucketMetadata>, org.jclouds.blobstore.domain.PageSet<? extends org.jclouds.blobstore.domain.StorageMetadata>> ignored on [EnhancedAnnotatedTypeImpl] public @Singleton class org.jclouds.s3.blobstore.functions.BucketsToStorageMetadata
Informações:   WELD-001125: Illegal bean type com.google.common.base.Function<org.jclouds.s3.domain.ListBucketResponse, org.jclouds.blobstore.domain.PageSet<? extends org.jclouds.blobstore.domain.StorageMetadata>> ignored on [EnhancedAnnotatedTypeImpl] public @Singleton class org.jclouds.s3.blobstore.functions.BucketToResourceList
Informações:   WELD-001125: Illegal bean type com.google.common.base.Supplier<java.util.Set<? extends org.jclouds.domain.Location>> ignored on [EnhancedAnnotatedTypeImpl] public @Singleton class org.jclouds.location.suppliers.all.RegionToProvider
Informações:   WELD-001125: Illegal bean type com.google.common.base.Supplier<java.util.Set<? extends org.jclouds.domain.Location>> ignored on [EnhancedAnnotatedTypeImpl] public @Singleton class org.jclouds.location.suppliers.all.ZoneToProvider
Informações:   WELD-001125: Illegal bean type com.google.common.base.Supplier<java.util.Set<? extends org.jclouds.domain.Location>> ignored on [EnhancedAnnotatedTypeImpl] public @Singleton class org.jclouds.location.suppliers.all.RegionToProviderOrJustProvider
Informações:   WELD-001125: Illegal bean type com.google.common.base.Supplier<java.util.Set<? extends org.jclouds.domain.Location>> ignored on [EnhancedAnnotatedTypeImpl] public @Singleton class org.jclouds.location.suppliers.all.ZoneToRegionToProviderOrJustProvider
Informações:   WELD-001125: Illegal bean type com.google.common.base.Supplier<java.util.Set<? extends org.jclouds.domain.Location>> ignored on [EnhancedAnnotatedTypeImpl] public @Singleton class org.jclouds.location.suppliers.all.JustProvider
Informações:   WELD-001125: Illegal bean type com.google.common.base.Function<java.util.Map<java.lang.String, java.lang.String>, java.util.Map<java.lang.String, ? extends org.jclouds.elasticstack.domain.Device>> ignored on [EnhancedAnnotatedTypeImpl] public @Singleton class org.jclouds.elasticstack.functions.MapToDevices
Informações:   WELD-001125: Illegal bean type com.google.common.base.Function<java.util.Map<java.lang.String, java.lang.String>, java.util.Map<java.lang.String, ? extends org.jclouds.elasticstack.domain.DriveMetrics>> ignored on [EnhancedAnnotatedTypeImpl] public @Singleton class org.jclouds.elasticstack.functions.MapToDriveMetrics
Informações:   WELD-001125: Illegal bean type com.google.common.base.Function<org.jclouds.atmos.domain.BoundedSet<? extends org.jclouds.atmos.domain.DirectoryEntry>, org.jclouds.blobstore.domain.PageSet<? extends org.jclouds.blobstore.domain.StorageMetadata>> ignored on [EnhancedAnnotatedTypeImpl] public @Singleton class org.jclouds.atmos.blobstore.functions.DirectoryEntryListToResourceMetadataList
Informações:   WELD-001125: Illegal bean type com.google.common.base.Supplier<java.util.Set<? extends org.jclouds.compute.domain.Hardware>> ignored on [EnhancedAnnotatedTypeImpl] public @Singleton class org.jclouds.aws.ec2.compute.suppliers.AWSEC2HardwareSupplier
Informações:   WELD-001125: Illegal bean type com.google.common.base.Function<java.lang.Iterable<? extends org.jclouds.cim.ResourceAllocationSettingData>, org.jclouds.compute.domain.HardwareBuilder> ignored on [EnhancedAnnotatedTypeImpl] public @Singleton class org.jclouds.cim.functions.HardwareBuilderFromResourceAllocations
Informações:   WELD-001125: Illegal bean type com.google.common.base.Function<java.lang.Iterable<java.util.Map.java.util.Map$Entry<java.lang.String, org.jclouds.ec2.options.DescribeImagesOptions>>, java.lang.Iterable<? extends org.jclouds.ec2.domain.Image>> ignored on [EnhancedAnnotatedTypeImpl] public @Singleton class org.jclouds.ec2.compute.strategy.DescribeImagesParallel
Informações:   WELD-001125: Illegal bean type com.google.common.base.Supplier<com.google.common.cache.LoadingCache<org.jclouds.ec2.compute.domain.RegionAndName, ? extends org.jclouds.compute.domain.Image>> ignored on [EnhancedAnnotatedTypeImpl] public @Singleton class org.jclouds.ec2.compute.suppliers.RegionAndNameToImageSupplier
Informações:   WELD-001125: Illegal bean type com.google.common.base.Supplier<java.util.Set<? extends org.jclouds.compute.domain.Image>> ignored on [EnhancedAnnotatedTypeImpl] public @Singleton class org.jclouds.ec2.compute.suppliers.EC2ImageSupplier
Informações:   WELD-001125: Illegal bean type com.google.common.base.Supplier<java.util.Set<? extends org.jclouds.compute.domain.Hardware>> ignored on [EnhancedAnnotatedTypeImpl] public @Singleton class org.jclouds.ec2.compute.suppliers.EC2HardwareSupplier
Informações:   WELD-001125: Illegal bean type com.google.common.base.Function<java.lang.Iterable<? extends org.jclouds.compute.domain.Image>, java.util.Map<org.jclouds.ec2.compute.domain.RegionAndName, ? extends org.jclouds.compute.domain.Image>> ignored on [EnhancedAnnotatedTypeImpl] public @Singleton class org.jclouds.ec2.compute.functions.ImagesToRegionAndIdMap
Informações:   WELD-001125: Illegal bean type com.google.common.base.Function<org.jclouds.azureblob.domain.ListBlobsResponse, org.jclouds.blobstore.domain.PageSet<? extends org.jclouds.blobstore.domain.StorageMetadata>> ignored on [EnhancedAnnotatedTypeImpl] public @Singleton class org.jclouds.azureblob.blobstore.functions.ListBlobsResponseToResourceList
Informações:   WELD-001125: Illegal bean type com.google.common.cache.CacheLoader<org.jclouds.openstack.nova.v2_0.domain.regionscoped.RegionAndId, java.lang.Iterable<? extends org.jclouds.openstack.nova.v2_0.domain.FloatingIP>> ignored on [EnhancedAnnotatedTypeImpl] public @Singleton class org.jclouds.openstack.nova.v2_0.compute.loaders.LoadFloatingIpsForInstance
Advertência:   Class class [Ljava.lang.String; has null package
Grave:   Exception during lifecycle processing
org.glassfish.deployment.common.DeploymentException: CDI definition failure:WELD-000071: Managed bean with a parameterized bean class must be @Dependent: class org.jclouds.openstack.swift.blobstore.SwiftBlobSigner
    at org.glassfish.weld.WeldDeployer.event(WeldDeployer.java:235)
    at org.glassfish.kernel.event.EventsImpl.send(EventsImpl.java:131)
    at org.glassfish.internal.data.ApplicationInfo.load(ApplicationInfo.java:329)
    at com.sun.enterprise.v3.server.ApplicationLifecycle.deploy(ApplicationLifecycle.java:497)
    at com.sun.enterprise.v3.server.ApplicationLifecycle.deploy(ApplicationLifecycle.java:220)
    at org.glassfish.deployment.admin.DeployCommand.execute(DeployCommand.java:487)
    at com.sun.enterprise.v3.admin.CommandRunnerImpl$2$1.run(CommandRunnerImpl.java:539)
    at com.sun.enterprise.v3.admin.CommandRunnerImpl$2$1.run(CommandRunnerImpl.java:535)
    at java.security.AccessController.doPrivileged(Native Method)
    at javax.security.auth.Subject.doAs(Subject.java:360)
    at com.sun.enterprise.v3.admin.CommandRunnerImpl$2.execute(CommandRunnerImpl.java:534)
    at com.sun.enterprise.v3.admin.CommandRunnerImpl$3.run(CommandRunnerImpl.java:565)
    at com.sun.enterprise.v3.admin.CommandRunnerImpl$3.run(CommandRunnerImpl.java:557)
    at java.security.AccessController.doPrivileged(Native Method)
    at javax.security.auth.Subject.doAs(Subject.java:360)
    at com.sun.enterprise.v3.admin.CommandRunnerImpl.doCommand(CommandRunnerImpl.java:556)
    at com.sun.enterprise.v3.admin.CommandRunnerImpl.doCommand(CommandRunnerImpl.java:1464)
    at com.sun.enterprise.v3.admin.CommandRunnerImpl.access$1300(CommandRunnerImpl.java:109)
    at com.sun.enterprise.v3.admin.CommandRunnerImpl$ExecutionContext.execute(CommandRunnerImpl.java:1846)
    at com.sun.enterprise.v3.admin.CommandRunnerImpl$ExecutionContext.execute(CommandRunnerImpl.java:1722)
    at com.sun.enterprise.v3.admin.AdminAdapter.doCommand(AdminAdapter.java:534)
    at com.sun.enterprise.v3.admin.AdminAdapter.onMissingResource(AdminAdapter.java:224)
    at org.glassfish.grizzly.http.server.StaticHttpHandlerBase.service(StaticHttpHandlerBase.java:189)
    at com.sun.enterprise.v3.services.impl.ContainerMapper$HttpHandlerCallable.call(ContainerMapper.java:466)
    at com.sun.enterprise.v3.services.impl.ContainerMapper.service(ContainerMapper.java:169)
    at org.glassfish.grizzly.http.server.HttpHandler.runService(HttpHandler.java:206)
    at org.glassfish.grizzly.http.server.HttpHandler.doHandle(HttpHandler.java:180)
    at org.glassfish.grizzly.http.server.HttpServerFilter.handleRead(HttpServerFilter.java:235)
    at org.glassfish.grizzly.filterchain.ExecutorResolver$9.execute(ExecutorResolver.java:119)
    at org.glassfish.grizzly.filterchain.DefaultFilterChain.executeFilter(DefaultFilterChain.java:284)
    at org.glassfish.grizzly.filterchain.DefaultFilterChain.executeChainPart(DefaultFilterChain.java:201)
    at org.glassfish.grizzly.filterchain.DefaultFilterChain.execute(DefaultFilterChain.java:133)
    at org.glassfish.grizzly.filterchain.DefaultFilterChain.process(DefaultFilterChain.java:112)
    at org.glassfish.grizzly.ProcessorExecutor.execute(ProcessorExecutor.java:77)
    at org.glassfish.grizzly.nio.transport.TCPNIOTransport.fireIOEvent(TCPNIOTransport.java:526)
    at org.glassfish.grizzly.strategies.AbstractIOStrategy.fireIOEvent(AbstractIOStrategy.java:112)
    at org.glassfish.grizzly.strategies.WorkerThreadIOStrategy.run0(WorkerThreadIOStrategy.java:117)
    at org.glassfish.grizzly.strategies.WorkerThreadIOStrategy.access$100(WorkerThreadIOStrategy.java:56)
    at org.glassfish.grizzly.strategies.WorkerThreadIOStrategy$WorkerThreadRunnable.run(WorkerThreadIOStrategy.java:137)
    at org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.doWork(AbstractThreadPool.java:591)
    at org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.run(AbstractThreadPool.java:571)
    at java.lang.Thread.run(Thread.java:745)
Caused by: org.jboss.weld.exceptions.DefinitionException: WELD-000071: Managed bean with a parameterized bean class must be @Dependent: class org.jclouds.openstack.swift.blobstore.SwiftBlobSigner
    at org.jboss.weld.bean.ManagedBean.checkType(ManagedBean.java:208)
    at org.jboss.weld.bean.AbstractBean.initializeAfterBeanDiscovery(AbstractBean.java:107)
    at org.jboss.weld.bean.ManagedBean.initializeAfterBeanDiscovery(ManagedBean.java:122)
    at org.jboss.weld.bootstrap.BeanDeployer.doAfterBeanDiscovery(BeanDeployer.java:352)
    at org.jboss.weld.bootstrap.BeanDeployment.afterBeanDiscovery(BeanDeployment.java:271)
    at org.jboss.weld.bootstrap.WeldStartup.deployBeans(WeldStartup.java:434)
    at org.jboss.weld.bootstrap.WeldBootstrap.deployBeans(WeldBootstrap.java:83)
    at org.glassfish.weld.WeldDeployer.event(WeldDeployer.java:223)
    ... 41 more

Grave:   Exception while loading the app
Grave:   Undeployment failed for context /PayaraJCloudsTest
Grave:   Exception while loading the app : CDI definition failure:WELD-000071: Managed bean with a parameterized bean class must be @Dependent: class org.jclouds.openstack.swift.blobstore.SwiftBlobSigner
org.jboss.weld.exceptions.DefinitionException: WELD-000071: Managed bean with a parameterized bean class must be @Dependent: class org.jclouds.openstack.swift.blobstore.SwiftBlobSigner
    at org.jboss.weld.bean.ManagedBean.checkType(ManagedBean.java:208)
    at org.jboss.weld.bean.AbstractBean.initializeAfterBeanDiscovery(AbstractBean.java:107)
    at org.jboss.weld.bean.ManagedBean.initializeAfterBeanDiscovery(ManagedBean.java:122)
    at org.jboss.weld.bootstrap.BeanDeployer.doAfterBeanDiscovery(BeanDeployer.java:352)
    at org.jboss.weld.bootstrap.BeanDeployment.afterBeanDiscovery(BeanDeployment.java:271)
    at org.jboss.weld.bootstrap.WeldStartup.deployBeans(WeldStartup.java:434)
    at org.jboss.weld.bootstrap.WeldBootstrap.deployBeans(WeldBootstrap.java:83)
    at org.glassfish.weld.WeldDeployer.event(WeldDeployer.java:223)
    at org.glassfish.kernel.event.EventsImpl.send(EventsImpl.java:131)
    at org.glassfish.internal.data.ApplicationInfo.load(ApplicationInfo.java:329)
    at com.sun.enterprise.v3.server.ApplicationLifecycle.deploy(ApplicationLifecycle.java:497)
    at com.sun.enterprise.v3.server.ApplicationLifecycle.deploy(ApplicationLifecycle.java:220)
    at org.glassfish.deployment.admin.DeployCommand.execute(DeployCommand.java:487)
    at com.sun.enterprise.v3.admin.CommandRunnerImpl$2$1.run(CommandRunnerImpl.java:539)
    at com.sun.enterprise.v3.admin.CommandRunnerImpl$2$1.run(CommandRunnerImpl.java:535)
    at java.security.AccessController.doPrivileged(Native Method)
    at javax.security.auth.Subject.doAs(Subject.java:360)
    at com.sun.enterprise.v3.admin.CommandRunnerImpl$2.execute(CommandRunnerImpl.java:534)
    at com.sun.enterprise.v3.admin.CommandRunnerImpl$3.run(CommandRunnerImpl.java:565)
    at com.sun.enterprise.v3.admin.CommandRunnerImpl$3.run(CommandRunnerImpl.java:557)
    at java.security.AccessController.doPrivileged(Native Method)
    at javax.security.auth.Subject.doAs(Subject.java:360)
    at com.sun.enterprise.v3.admin.CommandRunnerImpl.doCommand(CommandRunnerImpl.java:556)
    at com.sun.enterprise.v3.admin.CommandRunnerImpl.doCommand(CommandRunnerImpl.java:1464)
    at com.sun.enterprise.v3.admin.CommandRunnerImpl.access$1300(CommandRunnerImpl.java:109)
    at com.sun.enterprise.v3.admin.CommandRunnerImpl$ExecutionContext.execute(CommandRunnerImpl.java:1846)
    at com.sun.enterprise.v3.admin.CommandRunnerImpl$ExecutionContext.execute(CommandRunnerImpl.java:1722)
    at com.sun.enterprise.v3.admin.AdminAdapter.doCommand(AdminAdapter.java:534)
    at com.sun.enterprise.v3.admin.AdminAdapter.onMissingResource(AdminAdapter.java:224)
    at org.glassfish.grizzly.http.server.StaticHttpHandlerBase.service(StaticHttpHandlerBase.java:189)
    at com.sun.enterprise.v3.services.impl.ContainerMapper$HttpHandlerCallable.call(ContainerMapper.java:466)
    at com.sun.enterprise.v3.services.impl.ContainerMapper.service(ContainerMapper.java:169)
    at org.glassfish.grizzly.http.server.HttpHandler.runService(HttpHandler.java:206)
    at org.glassfish.grizzly.http.server.HttpHandler.doHandle(HttpHandler.java:180)
    at org.glassfish.grizzly.http.server.HttpServerFilter.handleRead(HttpServerFilter.java:235)
    at org.glassfish.grizzly.filterchain.ExecutorResolver$9.execute(ExecutorResolver.java:119)
    at org.glassfish.grizzly.filterchain.DefaultFilterChain.executeFilter(DefaultFilterChain.java:284)
    at org.glassfish.grizzly.filterchain.DefaultFilterChain.executeChainPart(DefaultFilterChain.java:201)
    at org.glassfish.grizzly.filterchain.DefaultFilterChain.execute(DefaultFilterChain.java:133)
    at org.glassfish.grizzly.filterchain.DefaultFilterChain.process(DefaultFilterChain.java:112)
    at org.glassfish.grizzly.ProcessorExecutor.execute(ProcessorExecutor.java:77)
    at org.glassfish.grizzly.nio.transport.TCPNIOTransport.fireIOEvent(TCPNIOTransport.java:526)
    at org.glassfish.grizzly.strategies.AbstractIOStrategy.fireIOEvent(AbstractIOStrategy.java:112)
    at org.glassfish.grizzly.strategies.WorkerThreadIOStrategy.run0(WorkerThreadIOStrategy.java:117)
    at org.glassfish.grizzly.strategies.WorkerThreadIOStrategy.access$100(WorkerThreadIOStrategy.java:56)
    at org.glassfish.grizzly.strategies.WorkerThreadIOStrategy$WorkerThreadRunnable.run(WorkerThreadIOStrategy.java:137)
    at org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.doWork(AbstractThreadPool.java:591)
    at org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.run(AbstractThreadPool.java:571)
    at java.lang.Thread.run(Thread.java:745)
```

I think it's a conflict between a dependence of JClouds and CDI used in Glassfish/Payara.

I tried to work with Glassfish 4.1 and 4.1.1 in the past, I could not. I am now using with Wildfly 10 and TomEE Plume.

Now I knew the excellent work of Payara team, so my hope is back. It would be possible to fix in any way?
## Steps to reproduce

Just create a Maven Web Java project with Netbeans, add JClouds dependency and try to deploy in Payara 163.

Sample https://github.com/douglasjunior/PayaraJCloudsTest
## Environment
- **Payara Version**: 4.1.1.163
- **Edition**: Full
- **JDK Version**: 8 Oracle
- **Operating System**:  Windows 10

@lprimak same problem. 

This line was added in `domain.xml`, that's right?

```xml
...
  <configs>
    <config name="server-config">
        ...
        <system-property name="fish.payara.classloading.delegate" value="false"></system-property>
```

Log:

```java
Launching GlassFish on Felix platform
out 30, 2016 2:17:17 PM com.sun.enterprise.glassfish.bootstrap.osgi.BundleProvisioner createBundleProvisioner
INFORMAÇÕES: Create bundle provisioner class = class com.sun.enterprise.glassfish.bootstrap.osgi.BundleProvisioner.
Registered com.sun.enterprise.glassfish.bootstrap.osgi.EmbeddedOSGiGlassFishRuntime@421cef37 in service registry.
out 30, 2016 2:17:28 PM com.sun.logging.LogDomains getResourceBundle
INFORMAÇÕES: Cannot find the resource bundle for the name com.sun.logging.enterprise.system.container.ejb for class org.glassfish.kernel.javaee.MEJBService using org.glassfish.main.core.javaee-kernel [134]
#!## LogManagerService.postConstruct : rootFolder=C:\Ferramentas\payara41\glassfish
#!## LogManagerService.postConstruct : templateDir=C:\Ferramentas\payara41\glassfish\lib\templates
#!## LogManagerService.postConstruct : src=C:\Ferramentas\payara41\glassfish\lib\templates\logging.properties
#!## LogManagerService.postConstruct : dest=C:\Ferramentas\payara41\glassfish\domains\domain1\config\logging.properties
Informações:   Running Payara Version: Payara Server  4.1.1.163 #badassfish (build 215)
Informações:   Server log file is using Formatter class: com.sun.enterprise.server.logging.ODLLogFormatter
Informações:   Registered org.glassfish.ha.store.adapter.cache.ShoalBackingStoreProxy for persistence-type = replicated in BackingStoreFactoryRegistry
Informações:   Registered fish.payara.ha.hazelcast.store.HazelcastBackingStoreFactoryProxy for persistence-type = hazelcast in BackingStoreFactoryRegistry
Informações:   Registered Hazelcast BackingStoreFactory with persistence-type = hazelcast
Informações:   Authorization Service has successfully initialized.
Informações:   Realm [admin-realm] of classtype [com.sun.enterprise.security.auth.realm.file.FileRealm] successfully created.
Informações:   Realm [file] of classtype [com.sun.enterprise.security.auth.realm.file.FileRealm] successfully created.
Informações:   Realm [certificate] of classtype [com.sun.enterprise.security.auth.realm.certificate.CertificateRealm] successfully created.
Informações:   Grizzly Framework 2.3.25 started in: 165ms - bound to [/0.0.0.0:8080]
Informações:   Grizzly Framework 2.3.25 started in: 7ms - bound to [/0.0.0.0:8181]
Informações:   Grizzly Framework 2.3.25 started in: 6ms - bound to [/0.0.0.0:4848]
Informações:   Grizzly Framework 2.3.25 started in: 2ms - bound to [/0.0.0.0:3700]
Informações:   Payara Server  4.1.1.163 #badassfish (215) startup time : Felix (11.843ms), startup services(1.455ms), total(13.298ms)
Informações:   Cleaning JarFileFactory Cache to prevent jar FD leaks
Informações:   Grizzly Framework 2.3.25 started in: 4ms - bound to [/0.0.0.0:7676]
Informações:   HV000001: Hibernate Validator 5.1.2.Final
Informações:   Grizzly Framework 2.3.25 started in: 9ms - bound to [/0.0.0.0:8181]
Informações:   Java security manager is disabled.
Informações:   Entering Security Startup Service.
Informações:   Loading policy provider com.sun.enterprise.security.provider.PolicyWrapper.
Informações:   Security Service(s) started successfully.
Informações:   Created HTTP listener http-listener-1 on host/port 0.0.0.0:8080
Informações:   Created HTTP listener http-listener-2 on host/port 0.0.0.0:8181
Informações:   Created HTTP listener admin-listener on host/port 0.0.0.0:4848
Informações:   Created virtual server server
Informações:   Created virtual server __asadmin
Informações:   JMXStartupService has started JMXConnector on JMXService URL service:jmx:rmi://douglas-hp:8686/jndi/rmi://douglas-hp:8686/jmxrmi
Informações:   Setting JAAS app name glassfish-web
Informações:   Virtual server server loaded default web module 
Informações:   Listening to REST requests at context: /management/domain.
Informações:   Registered com.sun.enterprise.glassfish.bootstrap.osgi.EmbeddedOSGiGlassFishImpl@2aa6311a as OSGi service registration: org.apache.felix.framework.ServiceRegistrationImpl@72725ee1.
Informações:   C:\Ferramentas\payara41\glassfish\domains\domain1\autodeploy\bundles does not exist, please create it.
Informações:   WELD-000900: 2.3.5 (Final)
Informações:   Inicializando Mojarra 2.2.13 ( 20160203-1910 unable to get svn info) para o contexto ''
Informações:   Loading application [__admingui] at [/]
Informações:   Loading application __admingui done in 5.399 ms
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.
Informações:   WELD-001125: Illegal bean type com.google.common.base.Function<org.jclouds.atmos.domain.BoundedSet<? extends org.jclouds.atmos.domain.DirectoryEntry>, org.jclouds.blobstore.domain.PageSet<? extends org.jclouds.blobstore.domain.StorageMetadata>> ignored on [EnhancedAnnotatedTypeImpl] public @Singleton class org.jclouds.atmos.blobstore.functions.DirectoryEntryListToResourceMetadataList
Informações:   WELD-001125: Illegal bean type com.google.common.base.Supplier<java.util.Set<? extends org.jclouds.compute.domain.Hardware>> ignored on [EnhancedAnnotatedTypeImpl] public @Singleton class org.jclouds.aws.ec2.compute.suppliers.AWSEC2HardwareSupplier
Informações:   WELD-001125: Illegal bean type com.google.common.base.Function<org.jclouds.blobstore.domain.PageSet<org.jclouds.openstack.swift.domain.ObjectInfo>, org.jclouds.blobstore.domain.PageSet<? extends org.jclouds.blobstore.domain.StorageMetadata>> ignored on [EnhancedAnnotatedTypeImpl] public @Singleton class org.jclouds.openstack.swift.blobstore.functions.ContainerToResourceList
Informações:   WELD-001125: Illegal bean type com.google.common.base.Function<java.lang.Iterable<java.util.Map.java.util.Map$Entry<java.lang.String, org.jclouds.ec2.options.DescribeImagesOptions>>, java.lang.Iterable<? extends org.jclouds.ec2.domain.Image>> ignored on [EnhancedAnnotatedTypeImpl] public @Singleton class org.jclouds.ec2.compute.strategy.DescribeImagesParallel
Informações:   WELD-001125: Illegal bean type com.google.common.base.Supplier<com.google.common.cache.LoadingCache<org.jclouds.ec2.compute.domain.RegionAndName, ? extends org.jclouds.compute.domain.Image>> ignored on [EnhancedAnnotatedTypeImpl] public @Singleton class org.jclouds.ec2.compute.suppliers.RegionAndNameToImageSupplier
Informações:   WELD-001125: Illegal bean type com.google.common.base.Supplier<java.util.Set<? extends org.jclouds.compute.domain.Image>> ignored on [EnhancedAnnotatedTypeImpl] public @Singleton class org.jclouds.ec2.compute.suppliers.EC2ImageSupplier
Informações:   WELD-001125: Illegal bean type com.google.common.base.Supplier<java.util.Set<? extends org.jclouds.compute.domain.Hardware>> ignored on [EnhancedAnnotatedTypeImpl] public @Singleton class org.jclouds.ec2.compute.suppliers.EC2HardwareSupplier
Informações:   WELD-001125: Illegal bean type com.google.common.base.Function<java.lang.Iterable<? extends org.jclouds.compute.domain.Image>, java.util.Map<org.jclouds.ec2.compute.domain.RegionAndName, ? extends org.jclouds.compute.domain.Image>> ignored on [EnhancedAnnotatedTypeImpl] public @Singleton class org.jclouds.ec2.compute.functions.ImagesToRegionAndIdMap
Informações:   WELD-001125: Illegal bean type com.google.common.base.Supplier<java.util.Set<? extends org.jclouds.compute.domain.Hardware>> ignored on [EnhancedAnnotatedTypeImpl] public @Singleton class org.jclouds.gogrid.compute.suppliers.GoGridHardwareSupplier
Informações:   WELD-001125: Illegal bean type com.google.common.base.Function<java.util.Set<org.jclouds.s3.domain.BucketMetadata>, org.jclouds.blobstore.domain.PageSet<? extends org.jclouds.blobstore.domain.StorageMetadata>> ignored on [EnhancedAnnotatedTypeImpl] public @Singleton class org.jclouds.s3.blobstore.functions.BucketsToStorageMetadata
Informações:   WELD-001125: Illegal bean type com.google.common.base.Function<org.jclouds.s3.domain.ListBucketResponse, org.jclouds.blobstore.domain.PageSet<? extends org.jclouds.blobstore.domain.StorageMetadata>> ignored on [EnhancedAnnotatedTypeImpl] public @Singleton class org.jclouds.s3.blobstore.functions.BucketToResourceList
Informações:   WELD-001125: Illegal bean type com.google.common.base.Function<org.jclouds.azureblob.domain.ListBlobsResponse, org.jclouds.blobstore.domain.PageSet<? extends org.jclouds.blobstore.domain.StorageMetadata>> ignored on [EnhancedAnnotatedTypeImpl] public @Singleton class org.jclouds.azureblob.blobstore.functions.ListBlobsResponseToResourceList
Informações:   WELD-001125: Illegal bean type com.google.common.base.Function<java.lang.Iterable<? extends org.jclouds.cim.ResourceAllocationSettingData>, org.jclouds.compute.domain.HardwareBuilder> ignored on [EnhancedAnnotatedTypeImpl] public @Singleton class org.jclouds.cim.functions.HardwareBuilderFromResourceAllocations
Informações:   WELD-001125: Illegal bean type com.google.common.base.Function<java.util.Map<java.lang.String, java.lang.String>, java.util.Map<java.lang.String, ? extends org.jclouds.elasticstack.domain.Device>> ignored on [EnhancedAnnotatedTypeImpl] public @Singleton class org.jclouds.elasticstack.functions.MapToDevices
Informações:   WELD-001125: Illegal bean type com.google.common.base.Function<java.util.Map<java.lang.String, java.lang.String>, java.util.Map<java.lang.String, ? extends org.jclouds.elasticstack.domain.DriveMetrics>> ignored on [EnhancedAnnotatedTypeImpl] public @Singleton class org.jclouds.elasticstack.functions.MapToDriveMetrics
Informações:   WELD-001125: Illegal bean type com.google.common.base.Supplier<java.util.Set<? extends org.jclouds.domain.Location>> ignored on [EnhancedAnnotatedTypeImpl] public @Singleton class org.jclouds.location.suppliers.all.RegionToProvider
Informações:   WELD-001125: Illegal bean type com.google.common.base.Supplier<java.util.Set<? extends org.jclouds.domain.Location>> ignored on [EnhancedAnnotatedTypeImpl] public @Singleton class org.jclouds.location.suppliers.all.ZoneToProvider
Informações:   WELD-001125: Illegal bean type com.google.common.base.Supplier<java.util.Set<? extends org.jclouds.domain.Location>> ignored on [EnhancedAnnotatedTypeImpl] public @Singleton class org.jclouds.location.suppliers.all.RegionToProviderOrJustProvider
Informações:   WELD-001125: Illegal bean type com.google.common.base.Supplier<java.util.Set<? extends org.jclouds.domain.Location>> ignored on [EnhancedAnnotatedTypeImpl] public @Singleton class org.jclouds.location.suppliers.all.ZoneToRegionToProviderOrJustProvider
Informações:   WELD-001125: Illegal bean type com.google.common.base.Supplier<java.util.Set<? extends org.jclouds.domain.Location>> ignored on [EnhancedAnnotatedTypeImpl] public @Singleton class org.jclouds.location.suppliers.all.JustProvider
Informações:   WELD-001125: Illegal bean type com.google.common.cache.CacheLoader<org.jclouds.openstack.nova.v2_0.domain.regionscoped.RegionAndId, java.lang.Iterable<? extends org.jclouds.openstack.nova.v2_0.domain.FloatingIP>> ignored on [EnhancedAnnotatedTypeImpl] public @Singleton class org.jclouds.openstack.nova.v2_0.compute.loaders.LoadFloatingIpsForInstance
Advertência:   Class class [Ljava.lang.String; has null package
Grave:   Exception during lifecycle processing
org.glassfish.deployment.common.DeploymentException: CDI definition failure:WELD-000071: Managed bean with a parameterized bean class must be @Dependent: class org.jclouds.openstack.swift.blobstore.SwiftBlobSigner
	at org.glassfish.weld.WeldDeployer.event(WeldDeployer.java:235)
	at org.glassfish.kernel.event.EventsImpl.send(EventsImpl.java:131)
	at org.glassfish.internal.data.ApplicationInfo.load(ApplicationInfo.java:329)
	at com.sun.enterprise.v3.server.ApplicationLifecycle.deploy(ApplicationLifecycle.java:497)
	at com.sun.enterprise.v3.server.ApplicationLifecycle.deploy(ApplicationLifecycle.java:220)
	at org.glassfish.deployment.admin.DeployCommand.execute(DeployCommand.java:487)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$2$1.run(CommandRunnerImpl.java:539)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$2$1.run(CommandRunnerImpl.java:535)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:360)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$2.execute(CommandRunnerImpl.java:534)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$3.run(CommandRunnerImpl.java:565)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$3.run(CommandRunnerImpl.java:557)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:360)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl.doCommand(CommandRunnerImpl.java:556)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl.doCommand(CommandRunnerImpl.java:1464)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl.access$1300(CommandRunnerImpl.java:109)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$ExecutionContext.execute(CommandRunnerImpl.java:1846)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$ExecutionContext.execute(CommandRunnerImpl.java:1722)
	at com.sun.enterprise.v3.admin.AdminAdapter.doCommand(AdminAdapter.java:534)
	at com.sun.enterprise.v3.admin.AdminAdapter.onMissingResource(AdminAdapter.java:224)
	at org.glassfish.grizzly.http.server.StaticHttpHandlerBase.service(StaticHttpHandlerBase.java:189)
	at com.sun.enterprise.v3.services.impl.ContainerMapper$HttpHandlerCallable.call(ContainerMapper.java:466)
	at com.sun.enterprise.v3.services.impl.ContainerMapper.service(ContainerMapper.java:169)
	at org.glassfish.grizzly.http.server.HttpHandler.runService(HttpHandler.java:206)
	at org.glassfish.grizzly.http.server.HttpHandler.doHandle(HttpHandler.java:180)
	at org.glassfish.grizzly.http.server.HttpServerFilter.handleRead(HttpServerFilter.java:235)
	at org.glassfish.grizzly.filterchain.ExecutorResolver$9.execute(ExecutorResolver.java:119)
	at org.glassfish.grizzly.filterchain.DefaultFilterChain.executeFilter(DefaultFilterChain.java:284)
	at org.glassfish.grizzly.filterchain.DefaultFilterChain.executeChainPart(DefaultFilterChain.java:201)
	at org.glassfish.grizzly.filterchain.DefaultFilterChain.execute(DefaultFilterChain.java:133)
	at org.glassfish.grizzly.filterchain.DefaultFilterChain.process(DefaultFilterChain.java:112)
	at org.glassfish.grizzly.ProcessorExecutor.execute(ProcessorExecutor.java:77)
	at org.glassfish.grizzly.nio.transport.TCPNIOTransport.fireIOEvent(TCPNIOTransport.java:526)
	at org.glassfish.grizzly.strategies.AbstractIOStrategy.fireIOEvent(AbstractIOStrategy.java:112)
	at org.glassfish.grizzly.strategies.WorkerThreadIOStrategy.run0(WorkerThreadIOStrategy.java:117)
	at org.glassfish.grizzly.strategies.WorkerThreadIOStrategy.access$100(WorkerThreadIOStrategy.java:56)
	at org.glassfish.grizzly.strategies.WorkerThreadIOStrategy$WorkerThreadRunnable.run(WorkerThreadIOStrategy.java:137)
	at org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.doWork(AbstractThreadPool.java:591)
	at org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.run(AbstractThreadPool.java:571)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.jboss.weld.exceptions.DefinitionException: WELD-000071: Managed bean with a parameterized bean class must be @Dependent: class org.jclouds.openstack.swift.blobstore.SwiftBlobSigner
	at org.jboss.weld.bean.ManagedBean.checkType(ManagedBean.java:208)
	at org.jboss.weld.bean.AbstractBean.initializeAfterBeanDiscovery(AbstractBean.java:107)
	at org.jboss.weld.bean.ManagedBean.initializeAfterBeanDiscovery(ManagedBean.java:122)
	at org.jboss.weld.bootstrap.BeanDeployer.doAfterBeanDiscovery(BeanDeployer.java:352)
	at org.jboss.weld.bootstrap.BeanDeployment.afterBeanDiscovery(BeanDeployment.java:271)
	at org.jboss.weld.bootstrap.WeldStartup.deployBeans(WeldStartup.java:434)
	at org.jboss.weld.bootstrap.WeldBootstrap.deployBeans(WeldBootstrap.java:83)
	at org.glassfish.weld.WeldDeployer.event(WeldDeployer.java:223)
	... 41 more

Grave:   Exception while loading the app
Grave:   Undeployment failed for context /PayaraJCloudsTest
Grave:   Exception while loading the app : CDI definition failure:WELD-000071: Managed bean with a parameterized bean class must be @Dependent: class org.jclouds.openstack.swift.blobstore.SwiftBlobSigner
org.jboss.weld.exceptions.DefinitionException: WELD-000071: Managed bean with a parameterized bean class must be @Dependent: class org.jclouds.openstack.swift.blobstore.SwiftBlobSigner
	at org.jboss.weld.bean.ManagedBean.checkType(ManagedBean.java:208)
	at org.jboss.weld.bean.AbstractBean.initializeAfterBeanDiscovery(AbstractBean.java:107)
	at org.jboss.weld.bean.ManagedBean.initializeAfterBeanDiscovery(ManagedBean.java:122)
	at org.jboss.weld.bootstrap.BeanDeployer.doAfterBeanDiscovery(BeanDeployer.java:352)
	at org.jboss.weld.bootstrap.BeanDeployment.afterBeanDiscovery(BeanDeployment.java:271)
	at org.jboss.weld.bootstrap.WeldStartup.deployBeans(WeldStartup.java:434)
	at org.jboss.weld.bootstrap.WeldBootstrap.deployBeans(WeldBootstrap.java:83)
	at org.glassfish.weld.WeldDeployer.event(WeldDeployer.java:223)
	at org.glassfish.kernel.event.EventsImpl.send(EventsImpl.java:131)
	at org.glassfish.internal.data.ApplicationInfo.load(ApplicationInfo.java:329)
	at com.sun.enterprise.v3.server.ApplicationLifecycle.deploy(ApplicationLifecycle.java:497)
	at com.sun.enterprise.v3.server.ApplicationLifecycle.deploy(ApplicationLifecycle.java:220)
	at org.glassfish.deployment.admin.DeployCommand.execute(DeployCommand.java:487)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$2$1.run(CommandRunnerImpl.java:539)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$2$1.run(CommandRunnerImpl.java:535)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:360)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$2.execute(CommandRunnerImpl.java:534)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$3.run(CommandRunnerImpl.java:565)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$3.run(CommandRunnerImpl.java:557)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:360)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl.doCommand(CommandRunnerImpl.java:556)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl.doCommand(CommandRunnerImpl.java:1464)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl.access$1300(CommandRunnerImpl.java:109)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$ExecutionContext.execute(CommandRunnerImpl.java:1846)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$ExecutionContext.execute(CommandRunnerImpl.java:1722)
	at com.sun.enterprise.v3.admin.AdminAdapter.doCommand(AdminAdapter.java:534)
	at com.sun.enterprise.v3.admin.AdminAdapter.onMissingResource(AdminAdapter.java:224)
	at org.glassfish.grizzly.http.server.StaticHttpHandlerBase.service(StaticHttpHandlerBase.java:189)
	at com.sun.enterprise.v3.services.impl.ContainerMapper$HttpHandlerCallable.call(ContainerMapper.java:466)
	at com.sun.enterprise.v3.services.impl.ContainerMapper.service(ContainerMapper.java:169)
	at org.glassfish.grizzly.http.server.HttpHandler.runService(HttpHandler.java:206)
	at org.glassfish.grizzly.http.server.HttpHandler.doHandle(HttpHandler.java:180)
	at org.glassfish.grizzly.http.server.HttpServerFilter.handleRead(HttpServerFilter.java:235)
	at org.glassfish.grizzly.filterchain.ExecutorResolver$9.execute(ExecutorResolver.java:119)
	at org.glassfish.grizzly.filterchain.DefaultFilterChain.executeFilter(DefaultFilterChain.java:284)
	at org.glassfish.grizzly.filterchain.DefaultFilterChain.executeChainPart(DefaultFilterChain.java:201)
	at org.glassfish.grizzly.filterchain.DefaultFilterChain.execute(DefaultFilterChain.java:133)
	at org.glassfish.grizzly.filterchain.DefaultFilterChain.process(DefaultFilterChain.java:112)
	at org.glassfish.grizzly.ProcessorExecutor.execute(ProcessorExecutor.java:77)
	at org.glassfish.grizzly.nio.transport.TCPNIOTransport.fireIOEvent(TCPNIOTransport.java:526)
	at org.glassfish.grizzly.strategies.AbstractIOStrategy.fireIOEvent(AbstractIOStrategy.java:112)
	at org.glassfish.grizzly.strategies.WorkerThreadIOStrategy.run0(WorkerThreadIOStrategy.java:117)
	at org.glassfish.grizzly.strategies.WorkerThreadIOStrategy.access$100(WorkerThreadIOStrategy.java:56)
	at org.glassfish.grizzly.strategies.WorkerThreadIOStrategy$WorkerThreadRunnable.run(WorkerThreadIOStrategy.java:137)
	at org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.doWork(AbstractThreadPool.java:591)
	at org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.run(AbstractThreadPool.java:571)
	at java.lang.Thread.run(Thread.java:745)

```

--- a/appserver/deployment/dol/src/main/java/com/sun/enterprise/deployment/Application.java
+++ b/appserver/deployment/dol/src/main/java/com/sun/enterprise/deployment/Application.java
@@ -60,7 +60,6 @@ import com.sun.enterprise.deployment.util.ComponentVisitor;
 import com.sun.enterprise.deployment.util.DOLUtils;
 import com.sun.enterprise.util.LocalStringManagerImpl;
 import com.sun.enterprise.util.StringUtils;
-import java.io.File;
 import java.net.URI;
 import java.net.URISyntaxException;
 import java.util.ArrayList;
@@ -1729,7 +1728,7 @@ public class Application extends CommonResourceBundleDescriptor
             input = input.replaceAll("(\\?|\\*)", ".$1");
             input = input.replaceFirst("\\.jar$", "");
             if(StringUtils.ok(libDir)) {
-                input = String.format("^%s%c%s(-.*)?\\.jar$", libDir, File.separatorChar, input);
+                input = String.format("^%s/%s(-.*)?\\.jar$", libDir, input);
             }
             return Pattern.compile(input, Pattern.CASE_INSENSITIVE);
         }

